web/src/app/icons/[icon]/opengraph-image.tsx

```diff
@@ -52,9 +52,9 @@ export default async function Image({ params }: { params: { icon: string } }) {
 				position: "relative",
 				fontFamily: "Inter, system-ui, sans-serif",
 				overflow: "hidden",
-				backgroundColor: "white",
+				backgroundColor: "#0f172a", // Dark background (slate-900)
```

Comment:
Why change the color ? It looks nice currently

Reply:
Doesn't fit the branding, imo.